genie/input_to_database.py

```diff
@@ -297,13 +242,26 @@ def validatefile(fileinfo,
 
     status_list = check_file_status['status_list']
     error_list = check_file_status['error_list']
-    filetype = get_filetype(syn, filepaths, center)
-    if check_file_status['to_validate']:
-
-        valid, message = _check_valid(
-            syn, filepaths, center, filetype, filenames,
-            oncotree_link, threads, testing)
+    # Need to figure out to how to remove this
 
+    filetype = validate.determine_filetype(
```

Comment:
catch error on this if you can't determine file type

Reply:
can remove this if you get `fileType` inside the `check_file_status` dictionary.

Reply:
I don't actually track filetype in the table.